tests/test_parser.py

```diff
@@ -581,8 +581,14 @@ def test_create_table_error(self, logger):
 
     def test_rename_table(self):
         self.assertEqual(
-            parse_one("ALTER TABLE foo RENAME TO bar").sql(),
-            "ALTER TABLE foo RENAME TO bar",
+            parse_one("ALTER TABLE table1 RENAME COLUMN IF EXISTS c1 TO c2").sql(),
+            "ALTER TABLE table1 RENAME COLUMN IF EXISTS c1 TO c2",
+        )
+
+    def test_alter_view_as(self):
+        self.assertEqual(
+            parse_one("ALTER VIEW foo AS SELECT a, b, c FROM bar").sql(),
+            "ALTER VIEW foo AS SELECT a, b, c FROM bar",
         )
```

Comment:
I'd suggest moving these two tests to `identity.sql`, not sure why `test_rename_table` was in `test_parser.py` to begin with.

Reply:
Sure. Will add more tests.

Reply:
moved.